pkgs/misc/themes/vertex/default.nix

```diff
@@ -1,33 +1,38 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, gtk3, pkgconfig }:
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, gnome3, gtk-engine-murrine }:
 
 stdenv.mkDerivation rec {
-  name = "theme-vertex-${version}";
-  version = "20160329";
+  name = "${pname}-${version}";
```

Comment:
My understanding is that the upstream name of the theme is "Vertex", it is only the repository name that is "vertex-theme". So my concern about naming the package `vertex-theme` is that we are naming it after the upstream repository name and not the actual upstream project name :-)

Also, there has been a discussion in https://github.com/NixOS/nixpkgs/pull/18928#discussion_r80413881 about renaming font packages to `fonts-XYZ` and if that goes through it might be reasonable to do something similar for various themes?

Sorry if I'm sounding a bit confused here, it has been a long day and my brain is getting a bit soft.